protocols/virtualdefund/virtualdefund_test.go

```diff
@@ -126,7 +129,8 @@ func testCrankAs(my ta.Actor) func(t *testing.T) {
 		testhelpers.Equals(t, waitingFor, WaitingForCompleteFinal)
 		signedByMe := state.NewSignedState(data.vFinal)
 		testhelpers.SignState(&signedByMe, &my.PrivateKey)
-		testhelpers.AssertStateSentToEveryone(t, se, signedByMe, my, allActors)
+		// TODO: Need to implement a helper that understands the virtual defund payload format
```

Comment:
Do this

Reply:
[d5c6f25](https://github.com/statechannels/go-nitro/pull/864/commits/d5c6f25fb34cc1d3ef311dbbe32f466560ebabc9)